docs/learn/fundamentals/transactions/operations-and-transactions.mdx

```diff
@@ -127,15 +127,15 @@ The source account must exist on the ledger.
 
 #### Fee
 
-The fee must be greater than or equal to the network minimum fee for the number of operations submitted as part of the transaction. This does not guarantee that the transaction will be applied, only that it is valid. In addition, the source account must be able to pay the fee specified. If multiple transactions are submitted but only a subset of them can be paid for, they are checked for validity in order of sequence number.
+The fee must be greater than or equal to the network minimum fee for the number of operations submitted as part of the transaction. This does not guarantee that the transaction will be applied, only that it is valid. In addition, the source account must be able to pay the fee specified.
 
 #### Fee-bump (if applicable)
 
 See Validity of a [Fee-Bump Transaction section](../../encyclopedia/transactions-specialized/fee-bump-transactions.mdx#validity-of-a-fee-bump-transaction)
 
 #### Sequence number
 
-The sequence number must be one greater than the sequence number stored in the source account entry when the transaction is applied unless sequence number preconditions are set. When checking the validity of multiple transactions with the same source account in a candidate transaction set, they must all be valid transactions and their sequence numbers must be offset by one. Then they are ordered and applied according to their sequence number.
+The sequence number must be one greater than the sequence number stored in the source account entry when the transaction is applied unless sequence number preconditions are set. An account can only have one transaction (and, therefore, one sequence number) consumed per candidate transaction set.
```

Comment:
I would say "per ledger" rather than "per candidate transaction set" since clients shouldn't need to understand SCP and how transactions sets are nominated to reason about how to submit transactions.

Reply:
The transaction lifecycle page can give a more nuanced explanation of the constraints.

Reply:
There's not much in this discussion currently, but improving the transaction lifecycle documentation is here: https://github.com/stellar/stellar-docs/discussions/1195... I'll link to this PR also.